.github/workflow_templates/e2e-daily.yml

```diff
@@ -0,0 +1,51 @@
+{!{- $enableWorkflowOnTestRepos := false -}!}
+{!{- $workflowName := "Daily e2e tests" -}!}
+name: '{!{ $workflowName }!}'
+on:
+  schedule:
+  - cron: '5 21 * * *'
```

Comment:
Let's change crontabs to something like:

1. 3am (or 4am/5am)
2. Weekly
3. From Monday to Friday

Something like this - `0 3 * * 1-5`. Now it is `Every day at 21:05.`

Reply:
changed